build.gradle

```diff
@@ -0,0 +1,36 @@
+// Apply the standard Java plugin
+plugins {
+    id 'java'
+    // Apply the Spotless plugin (using the same version as in the Kotlin DSL)
+    id 'com.diffplug.spotless' version '7.2.1'
```

Comment:
Requires minimum Java 11. And Java 8 won't accept the test program, of course becaue of the text blocks. Maybe that's ok as the runner can launch with Java 8.